url.bs

```diff
@@ -274,8 +274,8 @@ for further processing.
 <h3 id=host-miscellaneous>Host miscellaneous</h3>
 
 <p>A <dfn export>forbidden host code point</dfn> is U+0000 NULL, U+0009 TAB, U+000A LF, U+000D CR,
-U+0020 SPACE, U+0023 (#), U+0025 (%), U+002F (/), U+003A (:), U+003F (?), U+0040 (@), U+005B ([),
-U+005C (\), or U+005D (]).
+U+0020 SPACE, U+0023 (#), U+0025 (%), U+002F (/), U+003A (:), U+003C (<), U+003E (>), U+003F (?),
```

Comment:
```suggestion
U+0020 SPACE, U+0023 (#), U+0025 (%), U+002F (/), U+003A (:), U+003C (&lt;), U+003E (>), U+003F (?),
```

Reply:
If changing `<` to `$lt;` here, should I also change `>` to `&gt;`, to be consistent ?

Reply:
No that's not needed. HTML only requires it for `<` to disambiguate from a start tag. I guess maybe bikeshed does something so that it's not always needed, but I prefer keeping the source of the standard mostly HTML-compatible.